pypgatk/proteomics/openms.py

```diff
@@ -14,9 +17,29 @@ class OpenmsDataService(ParameterConfiguration):
   CONFIG_PEPTIDE_FDR_CUTOFF = "psm_pep_fdr_cutoff"
   CONFIG_PEPTIDE_CLASS_FDR_CUTOFF = "psm_pep_class_fdr_cutoff"
   CONFIG_PEPTIDE_GROUP_PREFIX = "peptide_groups_prefix"
-  CONFIG_PEPTIDE_APPLY_CLASS_FDR = "enable_class_fdr"
+  CONFIG_PEPTIDE_DISABLE_CLASS_FDR = "disable_class_fdr"
+  CONFIG_PEPTIDE_DISABLE_BAYESIAN_FDR = "disable_bayesian_class_fdr"
 
   def __init__(self, config_file, pipeline_arguments):
+
+    self._psm_df_index = "openms_psm_index"
+    self._psm_spectrum_reference = "spectrum_reference"
+    self._openms_exclude_columns = ['SpecId', 'Label', 'ScanNr', 'Peptide',
```

Comment:
this was very specific to some use case. here the columns you don't want to normalize or convert to numeric values need to go